RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -1210,4 +1210,94 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
   <data name="SourceControl_DeleteBranchButton" xml:space="preserve">
     <value>Zweig löschen</value>
   </data>
+  <data name="ExtractInteface_Caption" xml:space="preserve">
+    <value>Rubberduck - Extract Interface</value>
+  </data>
+  <data name="DeselectAll_Button" xml:space="preserve">
+    <value>Alles abwählen</value>
+  </data>
+  <data name="SelectAll_Button" xml:space="preserve">
+    <value>Alles auswählen</value>
+  </data>
+  <data name="ExtractInteface_TitleLabel" xml:space="preserve">
+    <value>Extract Interface</value>
```

Comment:
"ExtractInteface*"   That looks like my fault, you could change it in the base .resx as well.